packages/oid4vci-demo-frontend/src/configs/triall.json

```diff
@@ -173,7 +173,7 @@
   "routes": [
     {
       "id": "default",
-      "vpDefinitionId": "sphereonGuest",
+      "vpDefinitionId": "triall2023",
```

Comment:
not sure about this change. I've recently changed this to sphereon and the reason is that it doesn't make sense to ask for a credential that we're issuing

Reply:
This should be the definition id that is asking for the Sphereon Wallet Identity credential

Reply:
There is only a pd requesting a GuestCredential,  I created a new pd for the Sphereon Wallet Identity credential.